components/Forms/ProductForm.js

```diff
@@ -0,0 +1,219 @@
+import { defaultImageURL } from "@/public/images/defaultImageURL";
+import Link from "next/link";
+import { uid } from "uid";
+import {
+  StyledForm,
+  StyledTitleInput,
+  StyledLabel,
+  StyledInput,
+  StyledTextArea,
+} from "@/components/Forms";
+import {
+  StyledSubmitButton,
+  StyledCancelButton,
+  StyledButtonContainer,
+  StyledIconButton,
+} from "@/components/Buttons";
+import Icon from "@/components/Icons";
+import styled from "styled-components";
+import { useState } from "react";
+import Loading from "../Loading";
+
+const StyledImageUpload = styled.input`
+  display: none;
+`;
+
+const StyledUploadImageButton = styled(StyledIconButton)`
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  width: 130px;
+  justify-content: center;
+  background-color: var(--accentColor);
+  border-radius: 5px;
+  font-size: 12px;
+  color: var(--primaryBackgroundColor);
+  padding: 5px 10px;
+  box-shadow: 0px 1px 3px var(--primaryDarkColor);
+  &:hover {
+    background-color: var(--accentHoverColor);
+  }
+`;
+
+const StyledRemoveImageButton = styled(StyledUploadImageButton)`
+  left: 10px;
+  background-color: var(--dangerColor);
+  &:hover {
+    background-color: var(--dangerHoverColor);
+  }
+`;
+
+const StyledErrorMessage = styled.p`
+  color: var(--dangerColor);
+`;
+
+export default function ProductForm({
+  product = {},
+  stores,
+  currentImageURL,
+  onSubmit,
+  onSetCurrentImageURL,
+  onSetIsEdit,
+}) {
+  const [isUploading, setIsUploading] = useState(false);
+  const [isError, setIsError] = useState("");
```

Comment:
`isError` would suggest to me at this variable is a Boolean. 

Maybe it makes more sense to call it something like `errorMessage`?

Reply:
Oh absoluteley. It's called `errorMessage` now.